README.md

```diff
@@ -22,12 +22,13 @@ You can see the version history [here](RELEASE_NOTES.md).
 - FSharp.Core 4.3.1.0 (F# 3.1)
 - FSharp.Core 4.4.0.0 (F# 4.0)
 - FSharp.Core 4.4.1.0 (F# 4.1)
+- FSharp.Core 4.4.3.0 (F# 4.1 update)
 
 ## Supported platforms
 
-- .NET Framework 4.5
-- PCL profile 7 (.NET Framework 4.5, Windows 8)
-- PCL profile 259 (.NET Framework 4.5, Windows 8, Windows Phone 8.1, Windows Phone Silverlight 8)
+- .NET Framework 4.x
+- .NET Core App 2.x
+- .NET Standard 2.0
```

Comment:
Forgive my ignorance, but what's the difference between .NET Core App 2.x and .NET Standard 2.0?

Reply:
.NET Standard 2.0 is a big and large common library subset supported supported by .NET Framework 4.6+, Mono (including Xamarin Android/iOS) and .NET Core. It has a normal reflection model and has pretty much everything you actually need to build most libraries, and looks and feels like normal .NET 4.x programming for nearly all purposes.  All of FSharp.Data compiles without any `#if` when targeting .NET Standard 2.0.  It's basically what PCLs should always have been.

.NET Core App 2.0 (netcoreapp2.0) is the name of the programming model for a typical .NET Core application.  The programming model is basically the same as .NET Standard 2.0 but in theory you could have access to extra stuff that is not supported by .NET Standard (e.g. cross-platform ways of accessing .NET Core specifics about the .NET Core GC or something).